boards/arm/disco_l475_iot1/disco_l475_iot1.dts

```diff
@@ -116,6 +117,18 @@
 		spi-max-frequency = <2000000>;
 		label = "SPBTLE-RF";
 	};
+
+	wifi0: ism43362@0 {
+		compatible = "inventek,eswifi";
+		spi-max-frequency = <2000000>;
+		reg = <1>;
+		resetn-gpios = <&gpioe 8 0>;
+		boot0-gpios = <&gpiob 12 0>;
+		wakeup-gpios = <&gpiob 13 0>;
+		data-gpios = <&gpioe 1 0>;
+		csn-gpios = <&gpioe 0 0>;
```

Comment:
csn-gpios should be on spi3 node level. cs-gpios property should hold all the cs pins.

Reply:
Normally yes, however, the CSN pin seems to be more than a simple SPI chip select. It also delimits the command and data phase of the communication with the module. The module needs to be 'selected' as long as data phase is ongoing, which can take in theory several spi transfers (e.g. long tcp packet).  The current spi layer selects/unselects the chip for each transfer, which could cause the 'data' phase to be terminated prematurely.  So I propose to keep it under the device node for now. If it becomes a problem, I assume we could add to the spi layer the interface to manually lock for a specific device (e.g. spi_select_device/spi_unselect_device).

Reply:
There is already a configuration bit for holding the CS as long as you are doing transfer: SPI_HOLD_ON_CS
Once you are done, spi_release() has to be used (a bug was however recently discovered, and I sent a PR about it, not merged yet though.)

Apply @ydamigos comment anyway. (how spi API works has not much to with DTS).